app/src/main/java/ar/com/wolox/android/training/ui/training/main/tabs/news/NewsAdapter.kt

```diff
@@ -2,60 +2,60 @@ package ar.com.wolox.android.training.ui.training.main.tabs.news
 
 import android.net.Uri
 import android.view.LayoutInflater
+import android.view.View
 import android.view.ViewGroup
 import android.widget.ImageView
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import ar.com.wolox.android.R
 import ar.com.wolox.android.training.model.NewsItem
 import com.facebook.drawee.view.SimpleDraweeView
+import kotlinx.android.synthetic.main.news_item.view.*
 import org.ocpsoft.prettytime.PrettyTime
 import java.util.Locale
 
 class NewsAdapter(private val dataSet: List<NewsItem>, private val clickListener: (NewsItem) -> Unit) : RecyclerView.Adapter<NewsAdapter.NewsViewHolder>() {
 
-    class NewsViewHolder(inflater: LayoutInflater, parent: ViewGroup) :
-            RecyclerView.ViewHolder(inflater.inflate(R.layout.news_item, parent, false)) {
-        private var mUserView: TextView? = null
-        private var mNewsView: TextView? = null
-        private var mUserIcon: SimpleDraweeView? = null
-        private var mLikeBtn: ImageView? = null
-        private var mDate: TextView? = null
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): NewsViewHolder {
+        return NewsViewHolder(LayoutInflater.from(parent.context).inflate(R.layout.news_item, parent, false))
+    }
+
+    override fun onBindViewHolder(holder: NewsViewHolder, position: Int) {
+        val news: NewsItem = dataSet[position]
+        holder.bind(news, clickListener)
+    }
+
+    override fun getItemCount(): Int = dataSet.size
+
+    class NewsViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        private var vUsername: TextView? = null
```

Comment:
I'm almost sure you can do the assignations here.
```
private var vUsername = itemView.vUsername 
```
This will allow you to make it immutable and not null. 
Immutability is one of the most important characteristics of functional programming, and since Kotlin is functional-object, we should take care of it. 
Also, although Kotlin give us a good way to handle nulls, it shouldn't mean that we should make all nullable and then just check. If something is not null, we should try always to make it not null.